Makefile

```diff
@@ -10,15 +10,20 @@ MODULES := allometry assim.batch assim.sequential benchmark \
 				 data.mining data.remote emulator meta.analysis \
 				 photosynthesis priors rtm uncertainty
 
+SHINY := BenchmarkReport BrownDog Data-Ingest Elicitation Pecan.depend \
```

Comment:
if possible replace this with:

```
SHINY_DIRS := $(dir $(wildcard shiny/*/.))
SHINY := $(SHINY_DIRS:shiny/%/=%)
```

This will automatically generate the list of folders in the shiny folder, this way we don't have to remember to update the list in the Makefile

Reply:
Done in #2130. Guess we should eventually do the same for base/models/modules, once all their contents are ready to be included in the build...

Reply:
I don't disagree, but I did want to quickly mention that, IIRC, using `dir`, `wildcard`, and similar functions in Makefiles makes them "non-portable" for Windows users. I vaguely recall getting this flagged when running `R CMD check` on `PEcAnRTM`, which at some point used similar constructions in its Makevars file. Personally, I think bending over backwards to support Windows users isn't worth our time, at least at the level of PEcAn as a whole (individual packages should still try to work on Windows), especially now that the Docker workflow is maturing.

Reply:
I've been assuming that we threw out portability as soon as we started using secondary expansion. Am I confusing "portability between OSes" and "portability to non-GNU build systems"?

(also note that we already rely heavily on `wildcard` before this change, c.f. the bottom third of the Makefile)